.github/actions/templates/validateModuleDeployment/action.yml

```diff
@@ -279,14 +287,14 @@ runs:
 
     # [Deployment removal] task(s)
     # ----------------------------
-    - name: 'Remove [${{ inputs.templateFilePath }}] from parameters [${{ inputs.parameterFilePath }}]'
+    - name: 'Remove deployed resources'
```

Comment:
Ayn reason why we also remove the `templateFilePath` from the output? I was under the impression only the parameter file is the issue.
If that's correct, I'd strongly vote to keep the templateFilePath in as the new dependency approach essentially uses the templateFilePath to differentiate the different test cases :)

Reply:
Template file path is still in Job name